python/tk_hiero_export/sg_shot_processor.py

```diff
@@ -303,9 +303,17 @@ def startProcessing(self, exportItems):
         # tag app as first shot
         self.app.shot_count = 0
 
+        # need to temporarily monkey patch the internal hiero check so that our
+        # preview quicktime is generated. See the notes in the method being
+        # called for more info.
+        self._override_frame_server_check()
+
         # do the normal processing
         FnShotProcessor.ShotProcessor.startProcessing(self, exportItems)
 
+        # restore the monkey patched hiero method
+        self._restore_frame_server_check()
```

Comment:
The method calls have internal `try/except` blocks that should mask/log any issues. Errors will occur for older versions of hiero anyway. I mostly wanted to keep the hackery isolated to the methods. Let me know if you think it'd be better another way.